recipes/conan/meta.yaml

```diff
@@ -0,0 +1,116 @@
+{% set name = "conan" %}
+{% set version = "1.1.1" %}
+{% set sha256 = "3453edc61559e4cb2d12f7d4dcfec8860ccb644dc2155b1aaccf25b6022039cc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  entry_points:
+    - conan=conans.conan:run
+    - conan_server=conans.conan_server:run
+    - conan_build_info=conans.build_info.command:run
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
```

Comment:
Are all of these packages required to `build` this recipe? Typically when using `--single-version-externally-managed`, you only need `python` and `setuptools`, unless there is a `setup_requires` section in the `setup.py` or some packages get imported by that same script.

Reply:
Okay, that makes sense. I did not know this. Thanks!